src/main/scala/scorex/core/network/PeerConnectionHandler.scala

```diff
@@ -39,7 +39,7 @@ case class ConnectedPeer(socketAddress: InetSocketAddress,
   override def equals(obj: Any): Boolean =
     obj.cast[ConnectedPeer].exists(p => p.socketAddress == this.socketAddress && p.direction == this.direction)
 
-  override def toString = s"ConnectedPeer($socketAddress)"
+  override def toString: String = s"ConnectedPeer($socketAddress)"
```

Comment:
Recently, `direction` and `handshake` were added to `ConnectedPeer`. I wonder if we should update the `toString` method to show these new fields as well.

Reply:
@kushti, what do you think?

Reply:
I agree about direction, but not sure that handshake should be here.